examples/BUILD

```diff
@@ -27,6 +27,15 @@ cc_fuzz_test(
     srcs = ["empty_fuzz_test.cc"],
 )
 
+cc_fuzz_test(
+    name = "empty_fuzz_test_with_corpus",
+    srcs = ["empty_fuzz_test.cc"],
+    corpus = [
+        "corpus_1.txt",
+        "corpus_dir/corpus_2.txt",
```

Comment:
nit: small ask, can you demonstrate specifying a whole directory of files like `corpus_dir/**` with 1+ files? We often have a whole corpus, and want to specify that